Reference/Config/umbracoSettings/index.md

```diff
@@ -299,7 +299,7 @@ On new installs this set to true. When you are upgrading from a lower version th
 
 ## Security
 
-In the security section you have three options: **`<keepUserLoggedIn>`**, **`<allowPasswordReset>`** and **`<hideDisabledUsersInBackoffice>`**. Both settings are dealing with backoffice users.
+In the security section you have the following settings which deal with backoffice users.
```

Comment:
you have removed the `keepUserLoggedIn` while that is still a [valid configuration setting](https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Core/Configuration/UmbracoSettings/SecurityElement.cs#L7).

Reply:
The same remark for `hideDisabledUsersInBackoffice`

Reply:
Hi! I removed these because this line felt confusing!  They spoke of having "three options" but there were 4 listed beneath...but I guess it's talking about the 3 that are there by default when you install Umbraco?

I'll update it now to put them back in and update the wording a little :-)

Reply:
Actually, I've just realised the 3 that were originally there weren't the 3 that are included by default!  I'll add in all 5 into this sentence